java/server/src/org/openqa/grid/web/servlet/HubStatusServlet.java

```diff
@@ -109,35 +124,38 @@ protected void process(
       HttpServletRequest request,
       Map<String, Object> requestJSON) {
     Map<String, Object> res = new TreeMap<>();
-    res.put("success", true);
+    res.put(SUCCESS, true);
 
     try {
         List<String> keysToReturn = null;
 
-        if (request.getParameter("configuration") != null && !"".equals(request.getParameter("configuration"))) {
-          keysToReturn = Arrays.asList(request.getParameter("configuration").split(","));
-        } else if (requestJSON != null && requestJSON.containsKey("configuration")) {
+        if (request.getParameter(CONFIGURATION) != null && !"".equals(request.getParameter(CONFIGURATION))) {
```

Comment:
You can use `!Strings.isNullOrEmpty(request.getParameter(CONFIGURATION))` here for a slightly clearer check here (I realise that this wasn't done before)